src/components/common/DataTable.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div class="table-responsive">
+    <table :class="darkMode ? 'dark-table' : 'table'">
+      <thead>
+        <tr :class="darkMode ? 'table-header-dark' : 'table-header-light'">
+          <th
+            @click="setSortBy(column)"
+            v-for="(column, index) in fields"
+            :key="index"
```

Comment:
Using indexes as a key can lead to bugs, even if columns are pretty static I know I'm using the ```isEth``` getter, should that change it can throw things out of whack. Lets force the headers to include a ```key``` or ```id``` prop and go from that instead.

Reply:
added id prop to `ViewTableField` interface